evmd/app.go

```diff
@@ -486,6 +487,18 @@ func NewExampleApp(
 		&app.ConsensusParamsKeeper,
 		&app.Erc20Keeper,
 		tracer,
+	).WithStaticPrecompiles(
+		precompiletypes.DefaultStaticPrecompiles(
+			*app.StakingKeeper,
+			app.DistrKeeper,
+			app.BankKeeper,
```

Comment:
It changed from the `PreciseBankKeeper` to normal `BankKeeper` here, is that intentional?